client/app/components/packages/landuse-form/change-city-map.hbs

```diff
@@ -0,0 +1,367 @@
+        <label for="dcpEliminategradeopt">
+          Grade
+        </label>
+
+        <br>
+
+        <form.Field
+          @attribute="dcpEasement2"
+          @type="checkbox"
+          as |Checkbox|
+        >
+          <Checkbox>
+            Easement
+          </Checkbox>
+        </form.Field>
+
+      </Ui::Question>
+
+      <Ui::Question as |Q|>
+        <Q.Legend>
+          Does the project change any of the following? 
+          <small class="text-weight-normal">(select all that apply)</small>
+        </Q.Legend>
+
+        <Input
+          type="checkbox"
+          id="dcpChangestreetwidthopt"
+          data-test-checkbox="dcpChangestreetwidthopt"
+          checked={{if
+            (eq
+              form.data.dcpChangestreetwidthopt
+              (optionset 'landuseForm' 'dcpChangestreetwidthopt' 'code' 'Yes')
+            ) "checked"}}
+          onClick={{fn (mut form.data.dcpChangestreetwidthopt)
+            (if form.data.dcpChangestreetwidthopt
+              (optionset 'landuseForm' 'dcpChangestreetwidthopt' 'code' 'No')
+              (optionset 'landuseForm' 'dcpChangestreetwidthopt' 'code' 'Yes')
+            )
+          }}
+        />
+        <label for="dcpChangestreetwidthopt">
+          Street width
+        </label>
+
+        <br>
+
+        <Input
+          type="checkbox"
+          id="dcpChangestreetalignmentopt"
+          data-test-checkbox="dcpChangestreetalignmentopt"
+          checked={{if
+            (eq
+              form.data.dcpChangestreetalignmentopt
+              (optionset 'landuseForm' 'dcpChangestreetalignmentopt' 'code' 'Yes')
+            ) "checked"}}
+          onClick={{fn (mut form.data.dcpChangestreetalignmentopt)
+            (if form.data.dcpChangestreetalignmentopt
+              (optionset 'landuseForm' 'dcpChangestreetalignmentopt' 'code' 'No')
+              (optionset 'landuseForm' 'dcpChangestreetalignmentopt' 'code' 'Yes')
+            )
+          }}
+        />
+        <label for="dcpChangestreetalignmentopt">
+          Street alignment
+        </label>
+
+        <br>
+
+        <Input
+          type="checkbox"
+          id="dcpChangestreetgradeopt"
+          data-test-checkbox="dcpChangestreetgradeopt"
+          checked={{if
+            (eq
+              form.data.dcpChangestreetgradeopt
+              (optionset 'landuseForm' 'dcpChangestreetgradeopt' 'code' 'Yes')
+            ) "checked"}}
+          onClick={{fn (mut form.data.dcpChangestreetgradeopt)
+            (if form.data.dcpChangestreetgradeopt
+              (optionset 'landuseForm' 'dcpChangestreetgradeopt' 'code' 'No')
+              (optionset 'landuseForm' 'dcpChangestreetgradeopt' 'code' 'Yes')
+            )
+          }}
+        />
+        <label for="dcpChangestreetgradeopt">
+          Street grade
+        </label>
+
+        <br>
+
+        <form.Field
+          @attribute="dcpEasement3"
+          @type="checkbox"
+          as |Checkbox|
+        >
+          <Checkbox>
```

Comment:
Is this intentional?

Reply:
Oh.. does easement map to true/false unlike the others? CRM 🤣

Reply:
Yep! lolll the three easement fields in this file are all booleans whereas the other ones are integers (0, 1) -- so messy!